docs/user/04-metrics.md

```diff
@@ -405,7 +413,11 @@ This leads to the following limitations:
 
 ### Throughput
 
-The maximum throughput is 14K metric data points/sec ~= 50.000.000 metric data points/hour. If more data must be ingested, it can be refused. If a metric data endpoint emits more than 50.000 metric data points per scrape loop, all the data is refused by the agent.
+The metric gateway maximum throughput is 34K metric data points/sec ~= 122.000.000 metric data points/hour. If more data must be ingested, it can be refused.
+
+The metric agent maximum throughput is 14K metric data points/sec ~= 50.000.000 metric data points/hour. If more data must be ingested, it can be refused. If a metric data endpoint emits more than 50.000 metric data points per scrape loop, all the data is refused by the agent.
+
+> **NOTE:** By default metric gateway delivered with two instances to ensure load balancing work properly with gRPC, the metric source should be part of istio mesh and metric source should be configured with annotation `"traffic.sidecar.istio.io/includeOutboundPorts": "4317"`    
```

Comment:
```suggestion
> **NOTE:** By default, metric gateway is delivered with two instances to ensure load balancing works properly with gRPC. The metric source should be part of the Istio service mesh and metric source should be configured with the annotation `"traffic.sidecar.istio.io/includeOutboundPorts": "4317"`    
```

What does "should" mean - is this a recommendation? Is it optional? If it's not optional, use "must".